cornedbeef/src/main/java/com/swiftkey/cornedbeef/CoachMark.java

```diff
@@ -35,7 +35,7 @@
     public static final int NO_ANIMATION = 0;
     
     public interface OnDismissListener {
-        void onDismiss();
+        void onDismiss(boolean timeout);
```

Comment:
Something to think about...would it be better to have a new timeout listener, or have another method in this listener onTimeout() instead of this flag?